src/strategies/poap-with-weight/index.ts

```diff
@@ -0,0 +1,100 @@
+import { id } from '@ethersproject/hash';
+import { multicall, subgraphRequest } from '../../utils';
+import examplesFile from './examples.json';
+import aboutFile from './README.md';
+
+export const author = 'G2 & Torch';
```

Comment:
Change this to your GitHub ID
```suggestion
export const author = 'gawainb';
```